src/App.js

```diff
@@ -1,13 +1,24 @@
+import {
+  createBrowserRouter,
+  createRoutesFromElements,
+  Route,
+  RouterProvider,
+} from "react-router-dom";
+import RootLayout from "./layout/RootLayout";
 import BooksPage from "./pages/BooksPage";
 import CategoriesPage from "./pages/CategoriesPage";
 
+const router = createBrowserRouter(
+  createRoutesFromElements(
+    <Route path="/" element={<RootLayout />}>
+      <Route path="books" element={<BooksPage />} />
+      <Route path="categories" element={<CategoriesPage />} />
+    </Route>
+  )
+);
```

Comment:
- [ ] [Optional] I would suggest you please use `BooksPage` as the home page. When users visit the browser for the first time then they will see the list of books.